modules/nw-ovn-ipsec-north-south-enable.adoc

```diff
@@ -0,0 +1,138 @@
++
+--
+* `left_server.p12`: The certificate bundle for the IPsec endpoints
+* `ca.pem`: The certificate authority that you signed your certificates with
+--
+
+. Create a machine config to apply the IPsec configuration to your cluster:
+
+.. To add the IPsec configuration, create Butane config files for control plane and worker nodes with the following contents:
++
+[source,terminal,subs="attributes+"]
+----
+$ for role in master worker; do
+  cat >> "99-ipsec-$\{role}-endpoint-config.bu" <<-EOF
+  variant: openshift
+  version: {product-version}.0
+  metadata:
+    name: 99-$\{role}-import-certs-enable-svc-os-ext
+    labels:
+      machineconfiguration.openshift.io/role: $role
+  openshift:
+    extensions:
+      - ipsec
+  systemd:
+    units:
+    - name: ipsec-import.service
+      enabled: true
+      contents: |
+        [Unit]
+        Description=Import external certs into ipsec NSS
+        Before=ipsec.service
+      
+        [Service]
+        Type=oneshot
+        ExecStart=/usr/local/bin/ipsec-addcert.sh
+        RemainAfterExit=false
+        StandardOutput=journal
+
+        [Install]
+        WantedBy=multi-user.target
+    - name: ipsecenabler.service
+      enabled: true
+      contents: |
+        [Service]
+        Type=oneshot
+        ExecStart=systemctl enable --now ipsec.service
+
+        [Install]
+        WantedBy=multi-user.target
+  storage:
+    files:
+    - path: /etc/pki/certs/ca.pem
+      mode: 0400
+      overwrite: true
+      contents:
+        local: ca.pem
+    - path: /etc/pki/certs/left_server.p12
+      mode: 0400
+      overwrite: true
+      contents:
+        local: left_server.p12
+    - path: /usr/local/bin/ipsec-addcert.sh
+      mode: 0740
+      overwrite: true
+      contents:
+        inline: |
+          #!/bin/bash -e
+          echo "importing cert to NSS"
+          certutil -A -n "CA" -t "CT,C,C" -d /var/lib/ipsec/nss/ -i /etc/pki/certs/ca.pem
+          pk12util -W "" -i /etc/pki/certs/left_server.p12 -d /var/lib/ipsec/nss/
+          certutil -M -n "left_server" -t "u,u,u" -d /var/lib/ipsec/nss/
+EOF
+done
+----
+
+.. To transform the Butane files that you created in the previous step to a machine config, enter the following command:
++
+[source,terminal]
+----
+$ for role in master worker; do
+  butane 99-ipsec-${role}-endpoint-config.bu -o ./99-ipsec-$role-endpoint-config.yaml
+done
+----
+
+. To apply the machine configs to your cluster, enter the following command:
++
+[source,terminal]
+----
+$ for role in master worker; do
+  oc apply -f 99-ipsec-${role}-endpoint-config.yaml
+done
+----
+
+. As the MCO updates machines in each machine config pool, it reboots each node one by one. You must wait until all the nodes are updated. Check the machine config pool status by entering the following command:
```

Comment:
```suggestion
[IMPORTANT]
====
As the Machine Config Operator (MCO) updates machines in each machine config pool, it reboots each node one by one. You must wait until all the nodes are updated before [what?]. 
====

. Check the machine config pool status by entering the following command:
```
Didn't see MCO defined previously in the this module. 
Perhaps the "wait for it" should be an admonition?
Also don't know what we're waiting to do here; restart the cluster to apply settings, er? Can't hurt to specify.